cpp/velox/compute/WholeStageResultIterator.cc

```diff
@@ -367,6 +367,7 @@ std::unordered_map<std::string, std::string> WholeStageResultIterator::getQueryC
 
 #ifdef ENABLE_HDFS
 void WholeStageResultIterator::updateHdfsTokens() {
+  std::lock_guard lock{mutex};
```

Comment:
so the issue is due to these libhdfs3 functions are not thread-safe, or the `confMap_ ` ?

Reply:
libhdfs3 `hdfsSetDefautUserName`, but no specific method call was found